src/infrastructure/implementations/crux-messenger.ts

```diff
@@ -117,3 +117,14 @@ export const cruxPaymentProtocol: IMessageSchema[] = [{
             .required().min(36).max(36),
     }),
 }];
+
+export const cruxConenctProtocol: IMessageSchema[] = [{
+    messageType: "CONNECT_REQUEST",
+    schema: Joi.object({
+        publicKey: Joi.string()
+            .required(),
+
+        cruxId: Joi.string()
```

Comment:
Whose cruxId is this?

Reply:
will do it

Reply:
but actually kiski crux id hai ye? Sender has no Crux ID. Recipient has Crux ID but why does it need to be there in this message?

Reply:
actually its not required, mene ye disapprove wale case ke lie daal di thi hata deta hoon

Reply:
removed